pkg/packagekit/updates.jsx

```diff
@@ -543,58 +543,35 @@ class RestartServices extends React.Component {
     }
 }
 
-const ApplyUpdates = ({ transaction, onCancel, allowCancel, rebootAfter, setRebootAfter }) => {
-    const [percentage, setPercentage] = useState(0);
-    const [timeRemaining, setTimeRemaining] = useState(null);
-    // chronological list of { status: PK_STATUS_*, package: "name version" } events that happen during applying updates
-    const [actions, setActions] = useState([]);
-
-    const updateForProperties = (status, packageId) => {
-        // small timeout to avoid excessive overlaps from the next PackageKit progress signal
-        PK.call(transaction, "org.freedesktop.DBus.Properties", "GetAll", [PK.transactionInterface], { timeout: 500 })
-                .then(([props]) => {
-                    const percent = props.Percentage.v;
-                    let remain = -1;
-                    if ("RemainingTime" in props)
-                        remain = props.RemainingTime.v;
-
-                    const pfields = (packageId || props.LastPackage.v).split(";");
-                    // status: see PK_STATUS_* at https://github.com/PackageKit/PackageKit/blob/main/lib/packagekit-glib2/pk-enum.h
-                    setActions(prevActions => [...prevActions, {
-                        status: status || props.Status.v,
-                        package: pfields[0] + " " + pfields[1] + " (" + pfields[2] + ")"
-                    }]);
-
-                    const log = document.getElementById("update-log");
-                    let atBottom = false;
-                    if (log) {
-                        if (log.scrollHeight - log.clientHeight <= log.scrollTop + 2)
-                            atBottom = true;
-                    }
+const formatPackageId = packageId => {
+    const pfields = packageId.split(";");
+    return pfields[0] + " " + pfields[1] + " (" + pfields[2] + ")";
+};
 
-                    setPercentage(percent <= 100 ? percent : 0);
-                    setTimeRemaining(remain > 0 ? remain : null);
+// actions is a chronological list of { status, packageId } events that happen during applying updates
+// status: see PK_STATUS_* at https://github.com/PackageKit/PackageKit/blob/main/lib/packagekit-glib2/pk-enum.h
+const ApplyUpdates = ({ transactionProps, actions, onCancel, rebootAfter, setRebootAfter }) => {
+    const remain = transactionProps.RemainingTime
+        ? timeformat.distanceToNow(new Date().valueOf() + transactionProps.RemainingTime * 1000)
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18155-20230112-105327-48bdc90e-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)